e2e/js_image/runfiles.bzl

```diff
@@ -0,0 +1,129 @@
+"contains container helper functions for js_binary"
+
+load("@rules_pkg//:providers.bzl", "PackageFilegroupInfo", "PackageFilesInfo", "PackageSymlinkInfo")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+
+def _runfile_path(ctx, file, runfiles_dir):
+    path = file.short_path
+    if path.startswith(".."):
```

Comment:
nit: anything in aspect_bazel_lib that could/should be reused? I think we have a `to_manifest_path` which should be same as this helper

Reply:
good one. i'll replace it with the one from bazel-lib